hack/local/greeting_example_deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+registry=$1
```

Comment:
We could retrieve the minikube registry using this command

`kubectl get svc registry -n kube-system -ojsonpath='{.spec.clusterIP}'`

Reply:
I would leave it to the user calling the script.
And beware that there is the clusterIP for internal use and minikube IP for external use. This would be a bit more complex I guess.
I would say future evolution at some point, cause we may need it in GHA